project_generator/tools/iar.py

```diff
@@ -135,9 +135,17 @@ def _ewp_set_toolchain(self, ewp_dic, toolchain):
     def _ewp_set_name(self, ewp_dic, name):
         ewp_dic['project']['configuration']['name'] = name
 
-    def _eww_set_path(self, eww_dic, name):
+    def _eww_set_path_single_project(self, eww_dic, name):
```

Comment:
Should we have tool specific functions in the generic project class? Could this maybe be done in tool class?